src/main/java/com/cms/claimmanagement/insuranceCompany/surveyor/surveyReport/controller/ReportController.java

```diff
@@ -0,0 +1,31 @@
+package com.cms.claimmanagement.insuranceCompany.surveyor.surveyReport.controller;
+
+import com.cms.claimmanagement.insuranceCompany.surveyor.surveyReport.SurveyReportDTO;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/surveyReports")
```

Comment:
need to understand better, but maybe you could have /v1/survey/reports

Reply:
surveyReport is a document made by the surveyor, maybe surveyor/reports, what do you think?